swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+# from .
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = [] if inventory is None else inventory
+        self.inventory = inventory
```

Comment:
We can combine these two lines of code into one
```suggestion
        self.inventory = [] if inventory is None else inventory
```